main.js

```diff
@@ -151,14 +154,22 @@ function renderDetails(item) {
         createElement('p', { class: 'card-text item-description' }, [item.description]),
         createElement('p', { class: 'card-text item-details' }, [item.details]),
         createElement('p', { class: 'card-text item-origin' }, [item.origin]),
-        createElement('p', { class: 'card-text' }, [currencyFormat(item.price)])
+        createElement('p', { class: 'badge' }, [currencyFormat(item.price)]),
+        createElement('button', { class: 'btn btn-primary', id: 'add-to-cart' }, ['Add to Cart'])
       ])
     ])
   ])
 }
 
+function renderCart(cart) {
+  return createElement('div', { class: 'nav-item' }, [
+    'Cart: ',
+    createElement('span', { class: 'cart-number' }, [cart.items.length])
+  ])
+}
+
 function getItem(items, itemId) {
-  return items.filter(item => item.itemId === itemId)[0]
+  return items.filter((item) => item.itemId === itemId)[0]
```

Comment:
Try `Array.prototype.find`. If you have a long list, `filter` will do a _lot_ of unnecessary iterations.